paket.dependencies

```diff
@@ -0,0 +1,20 @@
+source https://www.nuget.org/api/v2/
+
+github fsprojects/FSharp.TypeProviders.StarterPack src/ProvidedTypes.fsi
+github fsprojects/FSharp.TypeProviders.StarterPack src/ProvidedTypes.fs
+github fsprojects/FSharp.TypeProviders.StarterPack src/DebugProvidedTypes.fs
+
+nuget FsCheck 0.9.2.0
+nuget FsCheck.Xunit 0.4.0.2
+nuget FSharp.Compiler.Service 0.0.44
+nuget FSharp.Data 2.0.7
+nuget FSharp.Formatting 2.4.8
+nuget Microsoft.AspNet.Razor 2.0.30506.0
```

Comment:
if you run `paket simplify` then this dependency will go away since it's only indirect.